collector/infiniband_linux.go

```diff
@@ -85,6 +85,38 @@ func NewInfiniBandCollector(logger log.Logger) (Collector, error) {
 		"port_receive_switch_relay_errors_total":     "Number of packets that could not be forwarded by the switch.",
 		"symbol_error_total":                         "Number of minor link errors detected on one or more physical lanes.",
 		"vl15_dropped_total":                         "Number of incoming VL15 packets dropped due to resource limitations.",
+
+		// https://enterprise-support.nvidia.com/s/article/understanding-mlx5-linux-counters-and-status-parameters
+		"duplicate_request_total":          "The number of received packets. A duplicate request is a request that had been previously executed.",
```

Comment:
duplicate_request**s**_total

Reply:
Thanks you for your review. change `duplicate_request_total ` to `duplicate_requests_total`